### PR TITLE
[zh] translate the getting started

### PR DESCRIPTION
**Pull request by zhihali**
Just translate the getting started files.

But some confusions here:

how can we translate "telemetry data", this word is a bit abstract if I translate it directly, I just translate it to 可观测性数据，I think it's make sense.

### CODE REVIEW

**Comment by svrnm** [comment]
@open-telemetry/docs-zh-approvers PTAL

File: content/zh/docs/getting-started/dev.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: 开发人员入门
+linkTitle: Dev
+default_lang_commit: e771c886739c4847b332b74f24b09d2769aab875
+---
+
+如果你符合以下条件，那么这个[入门指南](..)就是为你准备的：
+
+- 你是一名软件开发者。
+- 你的目标是通过编写代码来实现可观测性。
+- 你希望你的dependencies能自动生成可观测性数据。
```
Comment (windsonsea) [inline content/zh/docs/getting-started/dev.md:11]:
```suggestion
- 你希望你的依赖项能自动生成可观测性数据。
```

File: content/zh/docs/getting-started/dev.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+OpenTelemetry 可以帮到你！为了实现你的目标——自动为dependencies添加 instrumentation 以及使用我们的 API 手动为你自己的代码添加 instrumentation——我们建议你先学习以下概念：
```
Comment (windsonsea) [inline content/zh/docs/getting-started/dev.md:13]:
```suggestion
OpenTelemetry 可以帮到你！为了实现你的目标——自动为依赖项添加插桩以及使用我们的
API 手动为你自己的代码添加插桩——我们建议你先学习以下概念：
```

File: content/zh/docs/getting-started/dev.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- [什么是 OpenTelemetry？](../../what-is-opentelemetry/)
+- [如何为我的代码库添加 instrumentation？](../../concepts/instrumentation/code-based/)
```
Comment (windsonsea) [inline content/zh/docs/getting-started/dev.md:16]:
```suggestion
- [如何为我的代码库添加插桩？](../../concepts/instrumentation/code-based/)
```

File: content/zh/docs/getting-started/dev.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+如果你开发的是被其他软件作为依赖项使用的库、框架或中间件，我们建议你了解如何原生提供可观测性数据：
+
+- [如何为我的库添加原生 instrumentation？](../../concepts/instrumentation/libraries/)
```
Comment (windsonsea) [inline content/zh/docs/getting-started/dev.md:20]:
```suggestion
- [如何为我的库添加原生插桩？](../../concepts/instrumentation/libraries/)
```

File: content/zh/docs/getting-started/dev.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+接下来，你可以深入了解以下文档：
+你正在使用的[编程语言](../../languages/):
```
Comment (windsonsea) [inline content/zh/docs/getting-started/dev.md:23]:
```suggestion
接下来，你可以深入了解一些你会用到的[编程语言](../../languages/)文档：
```

File: content/zh/docs/getting-started/ops.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+title: 运维人员入门
+linkTitle: Ops
+default_lang_commit: e771c886739c4847b332b74f24b09d2769aab875
+---
+
+如果你符合以下条件，那么这个[入门指南](..)就是为你准备的：
+
+- 你在生产环境中运行一组应用程序。
+- 你的目标是从这些应用程序中获取可观测性数据（telemetry），而无需修改它们的代码。
+- 你想从多个服务中收集traces、metrics和logs，并将它们发送到你的可观测性后端。
```
Comment (windsonsea) [inline content/zh/docs/getting-started/ops.md:11]:
```suggestion
- 你想从多个服务中收集链路、指标和日志，并将它们发送到你的可观测性后端。
```

File: content/zh/docs/getting-started/ops.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+OpenTelemetry 可以帮到你！为了实现在不修改应用程序代码的情况下获取观测数据的目标，我们建议你学习以下内容：
+
+- [什么是 OpenTelemetry？](../../what-is-opentelemetry/)
+- [如何在不修改代码的情况下为应用程序添加观测性？](../../concepts/instrumentation/zero-code/)
+- [如何设置 Collector？](../../collector/)
+- [如何使用 OpenTelemetry Operator 在 Kubernetes 中实现自动化？](../../kubernetes/operator/)
+
+如果你想要实际体验这些功能，我们的官方演示项目提供了一套完整的[应用程序](/ecosystem/demo/)，非常适合实践学习。
```
Comment (windsonsea) [inline content/zh/docs/getting-started/ops.md:20]:
```suggestion
如果你想要实际体验这些功能，我们的[官方演示库提供了一套完整的应用](/ecosystem/demo/)，非常适合实践学习。
```

**Review by windsonsea — COMMENTED** [review]
Thanks, some suggestions for improvement.

File: content/zh/docs/getting-started/dev.md (hunk 2 above)
Comment (zhihali) [reply content/zh/docs/getting-started/dev.md:13]: 我有些疑惑这个插桩，还有依赖如果直译的话是否有点抽象？

**Review by windsonsea — APPROVED** [review]
/lgtm

**Comment by zhihali** [comment]
@windsonsea hi 老哥，我想咨询一波，一般来说翻译多少开个PR比较合适？我觉得一节的内容开个PR比较容易review，但是有些时候感觉有点短，你觉得呢？

**Comment by windsonsea** [comment]
小步快走，一个文件一个PR比较好

**Review by my-git9 — APPROVED** [review]

**Pull request merged**